src/elements/category/src/index.tsx

```diff
@@ -48,7 +48,7 @@ const Category: React.FC<ListProps> = ({
           </div>
         )}
         <Row>
-          <Col span={contentSpan} sx={{ marginBottom: 0 }}>
+          <Col span={contentSpan} sx={{ marginBottom: [0, 0] }}>
```

Comment:
do you need to define 2 0's in the array? I thought if there is only one value it applies that to all breakpoints

Reply:
yeah it does, but because there's a `marginBottom: ['something', 'something']` on the `<Col>` component, the media query generated by that makes it more specific than the margin bottom originally added here